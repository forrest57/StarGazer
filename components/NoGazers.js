import { StyleSheet, Text, View } from 'react-native'

import { FontAwesome5 } from '@expo/vector-icons'
import tw from 'twrnc'

export default NoGazers = () => (
  <View style={tw`w-full h-full flex items-center justify-start pt-15`}>
    <FontAwesome5 name='sad-tear' size={150} color='#4078c0' />
    <Text style={styles.text}>No Stargazers?</Text>
  </View>
)
const styles = StyleSheet.create({
  text: {
    fontSize: 35,
    color: 'gray',
    fontWeight: 'bold',
    fontStyle: 'italic',
  },
})
