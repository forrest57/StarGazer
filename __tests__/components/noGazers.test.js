import NoGazers from '../../components/NoGazers'
import renderer from 'react-test-renderer'

describe('<NoGazers/>', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<NoGazers />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
