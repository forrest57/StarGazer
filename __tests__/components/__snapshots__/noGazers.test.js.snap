// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NoGazers/> renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "bottom": 0,
      "justifyContent": "center",
      "left": 0,
      "position": "absolute",
      "right": 0,
      "top": 0,
    }
  }
  textId="noGazers"
>
  <Text />
  <Text
    style={
      Object {
        "color": "gray",
        "fontSize": 35,
        "fontStyle": "italic",
        "fontWeight": "bold",
      }
    }
  >
    No Stargazers?
  </Text>
</View>
`;
